bylaws/2-cf-membership.md

```diff
@@ -20,22 +16,28 @@ The following section outlines the different types of membership and the process
 
 A **project** within the CF represents collaborative innovation in open source technology. These projects are dynamic collectives of individual contributors, functioning as self-organizing units rather than formal legal entities.[^1]
 
-**Flexible and Inclusive Structure:** Projects can vary in size and scope, ranging from individual repositories to entire collections of repositories. Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+**Flexible and Inclusive Structure:** Projects vary in size and scope&mdash;a project might have a single repository, or it might encompass multiple repositories.
+Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+While the CF uses GitHub as the primary platform for collaboration, projects may be hosted on other platforms as well.
 
 [^1]: Contributors may be affiliated with various organizations or companies, but they contribute as individuals.
 
-## Project Representation
+### Project Representation
 
 **Diverse Voices, Unified Decision-Making:** Each project within the Commonhaus Foundation (CF) has its unique perspective, and its own legitimate interests in decisions taken by the foundation as a whole.
 To ensure that every project has a say in the direction of the foundation, each project appoints a representative to the [Extended Governance Committee (EGC)][egc].
-The representative is chosen according to the project's own governance practices.
-The representative is not required to have or maintain general membership of the foundation.
 
-**Dedicated Representation:** If an EGC representative is elected to the [CF Council (CFC)][cfc], we recommend appointing an additional representative for the EGC to ensure project-specific needs are represented.
+The EGC representative:
+
+- is chosen according to the project's own governance practices,
+- is not required to have or maintain general membership in the foundation, but
+- must have a GitHub account to participate in CF [discussions][discussions] and [votes][voting].
```

Comment:
I know it is minor thing but should we leave out explicit technology choices of memberships in the bylaws and have that in policy/implementatin docs instead?

Speaking out of experience that Eclipse had to do tedious updates to bylaws that requires extra level of voting as it had bugzilla and other superhip tech in their bylaws.

Reply:
In this case, there were questions about whether or not an account would be required. 

I am working on the EGC member checklist separately, so I could move that there.